internal/api/v1/router.go

```diff
@@ -131,13 +131,16 @@ var Routes = routes.NamedRoutes{
 	"ConfigurationUpdate":  patch("/namespaces/:namespace/configurations/:configuration", errorHandler(configuration.Controller{}.Update)),
 	"ConfigurationReplace": put("/namespaces/:namespace/configurations/:configuration", errorHandler(configuration.Controller{}.Replace)),
 
+	// Service Catalog
+	"ServiceCatalog":     get("/catalogservices", errorHandler(service.Controller{}.Catalog)),
+	"ServiceCatalogShow": get("/catalogservices/:catalogservice", errorHandler(service.Controller{}.CatalogShow)),
+
 	// Services
-	"ServiceCatalog":     get("/services", errorHandler(service.Controller{}.Catalog)),
-	"ServiceCatalogShow": get("/services/:catalogservice", errorHandler(service.Controller{}.CatalogShow)),
-	"ServiceCreate":      post("/namespaces/:namespace/services", errorHandler(service.Controller{}.Create)),
-	"ServiceList":        get("/namespaces/:namespace/services", errorHandler(service.Controller{}.List)),
-	"ServiceShow":        get("/namespaces/:namespace/services/:service", errorHandler(service.Controller{}.Show)),
-	"ServiceDelete":      delete("/namespaces/:namespace/services/:service", errorHandler(service.Controller{}.Delete)),
+	"AllServices":   get("/services", errorHandler(service.Controller{}.FullIndex)),
+	"ServiceCreate": post("/namespaces/:namespace/services", errorHandler(service.Controller{}.Create)),
+	"ServiceList":   get("/namespaces/:namespace/services", errorHandler(service.Controller{}.List)),
+	"ServiceShow":   get("/namespaces/:namespace/services/:service", errorHandler(service.Controller{}.Show)),
+	"ServiceDelete": delete("/namespaces/:namespace/services/:service", errorHandler(service.Controller{}.Delete)),
```

Comment:
General side note, need a ticket that the service show/delete commands do not have auto-completion, and for this `Match` endpoints are needed here, similar to what we have for other resources.

Reply:
Done! https://github.com/epinio/epinio/issues/1476 :slightly_smiling_face: